.travis.yml

```diff
@@ -11,33 +11,33 @@ stages:
 # 3. Then run tests compiled via dartdevc and dart2js.
 jobs:
     include:
-    # mockito tests cannot run on stable until stable is >= 2.9.0, as "2.9.0"
+    # mockito tests cannot run on stable until stable is >= 2.10.0, as "2.10.0"
```

Comment:
Note that you could likely get around this with a dependency override?

Reply:
Ah, neat trick. I'll try this in a follow-up.